en/ros/external_position_estimation.md

```diff
@@ -1,114 +1,220 @@
+  The `geometry_msgs/PoseStamped` topic is most common as MoCap doesn't usually have associated covariances to the data.
+- If you get data through a `nav_msgs/Odometry` ROS message then you will need to remap it to `/mavros/odometry/odom`.
+- TBD from `/vrpn_client_node/<rigid_body_name>/pose` to `/mavros/vision_pose/pose`. 
 
-## Asserting on Reference Frames
 
-This section shows how to setup the system with the proper reference frames. There are various representations but we will use two of them: ENU and NED. 
+## Asserting on Reference Frames 
 
-* ENU has a ground-fixed frame where *x* axis points East, *y* points North and *z* up. Robot frame is *x* towards the front, *z* up and *y* accordingly.
+This section shows how to setup the system with the proper reference frames. 
+There are various representations but we will use two of them: ENU and NED. 
 
-* NED has *x* towards North, *y* East and *z* down. Robot frame is *x* towards the front, *z* down and *y* accordingly.
+* ENU has a ground-fixed frame where *x* axis points East, *y* points North and *z* up.
+  Robot body frame is *x* towards the front, *z* up and *y* accordingly.
+* NED has *x* towards North, *y* East and *z* down.
+  Robot body frame is *x* towards the front, *z* down and *y* accordingly.
 
 Frames are shown in the image below: NED on the left while ENU on the right.
 
 ![Reference frames](../../assets/lpe/ref_frames.png)
 
-With the external heading estimation, however, magnetic North is ignored and faked with a vector corresponding to world *x* axis (which can be placed freely at mocap calibration); yaw angle will be given with respect to local *x*.
+With the external heading estimation, however, magnetic North is ignored and faked with a vector corresponding to world *x* axis (which can be placed freely at MoCap calibration); 
+yaw angle will be given with respect to local *x*.
 
-> **Info** When creating the rigid body in the mocap software, remember to first align the robot's local *x* axis with the world *x* axis otherwise yaw estimation will have an initial offset.
+> **Note** When creating the rigid body in the MoCap software, remember to first align the robot's local *x* axis with the world *x* axis otherwise yaw estimation will have an initial offset.
 
-### Using Mavros
+### Using MAVROS
 
-With MAVROS this operation is straightforward. ROS uses ENU frames as convention, therefore position feedback must be provided in ENU. If you have an Optitrack system you can use [mocap_optitrack](https://github.com/ros-drivers/mocap_optitrack) node which streams the object pose on a ROS topic already in ENU. With a remapping you can directly publish it on `mocap_pose_estimate` as it is without any transformation and mavros will take care of NED conversions.
+With MAVROS this operation is straightforward. 
+ROS uses ENU frames as convention, therefore position feedback must be provided in ENU. 
```

Comment:
I think perhaps this is good generic advice for a system integrating with ROS?

Reply:
And basically all this is specific to Optitrack and should live under that?

Reply:
This is not specific to optitrack. It's applied to vision frameworks as well.